### PR TITLE
Altered Raphtory deployment function names to clarify their purpose

### PR DESCRIPTION
**Pull request by miratepuffin**
After a couple of discussions we have decided to update the function names of the `Raphtory` object to clarify what they are actually doing. These changes are the following:

- `Raphtory.stream()` will remain the same as its fairly clear what this is doing.
- `Raphtory.batchLoad()` has been updated to `.load()` as the batch component was deemed confusing. This was also to make it clearer in the tutorial the we are just loading a graph off of disk.
- `Raphtory.deployedGraph()` has been updated to `.connect()` as deployedGraph didn't sound like an action -- and to match the `disconnect` function used to close the connection.

All component construction functions have also been privatised as there is no need for them to be accessible by the end user.

All documentation and examples have been updated accordingly (I hope 😄)

### CODE REVIEW

File: docs/source/Analysis/queries.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 To run your implemented algorithm or any of the algorithms included in Raphtory (both [Generic](com.raphtory.algorithms.generic) and [Temporal](com.raphtory.algorithms.temporal)), you must submit them to the graph. We can again use the [Lord of the Rings
 graph](../Ingestion/sprouter.md) and the [degrees of separation algorithm](LOTR_six_degrees.md) to illustrate the query API.
 
-When running queries, our starting point is always the {scaladoc}`com.raphtory.algorithms.api.TemporalGraph` created from a call to `Raphtory.batchLoad()` or `Raphtory.stream()`. This contains the full history of your data over its lifetime. From this point, the overall process to get things done is as follows: 
+When running queries, our starting point is always the {scaladoc}`com.raphtory.algorithms.api.TemporalGraph` created from a call to `Raphtory.load()` or `Raphtory.stream()`. This contains the full history of your data over its lifetime. From this point, the overall process to get things done is as follows:
```
Comment (ljeub-pometry) [inline docs/source/Analysis/queries.md:6]: Should this mention the `Raphtory.connect()` option?

File: docs/source/Deployment/baremetalsingle.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+The difference between the two from an analysis perspective is that `load` will block any submitted queries until the data has finished ingesting - allowing it to handle completely out of order data. On the other hand when using `stream` it is assumed new data is continuously arriving (in roughly chronological order) which Raphtory handles with a watermarking heuristic to decide what time is safe to analyse across all the partitions. Therefore, queries where the perspective time   fully ingested and synchronised allowed to progress and which should be blocked until the time they are set to run at has arrived and has been synchronised.
```
Comment (ljeub-pometry) [inline docs/source/Deployment/baremetalsingle.md:3]: Something got lost in the second sentence (didn't happen in this pull request). Needs to be fixed.

**Review by ljeub-pometry — APPROVED** [review]
Looking good, I like the new names.

I noticed some unrelated issues with the docs which may be worth fixing either quickly here or as a separate pull request.

File: docs/source/Deployment/baremetalsingle.md (hunk 2 above)
Comment (miratepuffin) [reply docs/source/Deployment/baremetalsingle.md:3]: Oh yeah what an earth happened there! I shall fix it

**Pull request merged**